src/components/POLine/Item/ItemView.js

```diff
@@ -20,15 +21,21 @@ class ItemView extends Component {
 
   render() {
     const { poLineDetails, identifierTypes } = this.props;
+    const instanceId = get(poLineDetails, 'instanceId');
```

Comment:
```suggestion
    const instanceId = get(poLineDetails, 'instanceId');
    const title = get(poLineDetails, 'title');
    const titleValue = instanceId
      ? <Link to={`/inventory/view/${instanceId}`}>title</Link>
      : title;
```
and below in `return (` you can just put `{titleValue}`

Reply:
done

Reply:
done